### PR TITLE
fix: update glyphs to use currentcolor

### LINKED ISSUE #2260: Update glyphs to use currentcolor
**What feature are you requesting?**
In many places we keep `color` and `fill` synchronized because svg icons generally need a fill that's the same as related text's foreground color. Set the fill of all svg paths to [currentcolor](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentColor) and remove the fill attributes in the stylesheets.

**What would be the benefit of adding this feature?**
We can simplify our stylesheets, including high contrast overrides.

### PR DESCRIPTION
**Pull request by brookdozer**
# Description
- Modify css for svg fill to use currentcolor when the svg should default to the text color
- Align color fill for menu-item, flipper, text-field, tree-item, tab, and slider into alignment with anchor and button
- Modify text-field input to inherit color from root and root fill to use currentcolor

## Motivation & context

Align SVG color fill with fast-anchor and fast-button
https://github.com/microsoft/fast/issues/2260

Closes #2260  

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [x] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

**Comment by EisenbergEffect** [comment]
Anyone know why GitHub would add Bill as a PR reviewer? That seems random...

**Comment by EisenbergEffect** [comment]
Removed incorrect Bill 🤣 Will add correct Bill shortly.

**Comment by brookdozer** [comment]
My mistake: should have been williamw2. Fixed

File: packages/web-components/fast-components/src/flipper/flipper.styles.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ export const FlipperStyles = css`
         align-items: center;
         margin: 0;
         position: relative;
-        fill: ${accentForegroundCutRestBehavior.var};
+        fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/flipper/flipper.styles.ts:30]:
```suggestion
        fill: currentcolor;
```

File: packages/web-components/fast-components/src/flipper/flipper.styles.ts [hunk 2]
```diff
@@ -61,7 +61,7 @@ export const FlipperStyles = css`
     :host(.disabled) {
         opacity: var(--disabled-opacity);
         cursor: ${disabledCursor};
-        fill: ${neutralForegroundRestBehavior.var};
+        fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/flipper/flipper.styles.ts:64]:
```suggestion
        fill: currentcolor;
```

File: packages/web-components/fast-components/src/flipper/flipper.styles.ts [hunk 3]
```diff
@@ -114,7 +114,7 @@ export const FlipperStyles = css`
             :host .next,
             :host .previous {
                 color: ${SystemColors.ButtonText};
-                fill: ${SystemColors.ButtonText};
+                fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/flipper/flipper.styles.ts:117]:
```suggestion
                fill: currentcolor;
```

File: packages/web-components/fast-components/src/flipper/flipper.styles.ts [hunk 4]
```diff
@@ -130,7 +130,7 @@ export const FlipperStyles = css`
             :host(:hover) .previous {
                 forced-color-adjust: none;
                 color: ${SystemColors.HighlightText};
-                fill: ${SystemColors.HighlightText};
+                fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/flipper/flipper.styles.ts:133]:
```suggestion
                fill: currentcolor;
```

File: packages/web-components/fast-components/src/menu-item/menu-item.styles.ts [hunk 5]
```diff
@@ -31,7 +31,7 @@ export const MenuItemStyles = css`
         white-space: nowrap;
         overflow: hidden;
         color: ${neutralForegroundRestBehavior.var};
-        fill: ${neutralForegroundRestBehavior.var};
+        fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/menu-item/menu-item.styles.ts:34]:
```suggestion
        fill: currentcolor;
```

File: packages/web-components/fast-components/src/menu-item/menu-item.styles.ts [hunk 6]
```diff
@@ -65,7 +65,7 @@ export const MenuItemStyles = css`
 
     :host(.disabled:hover) {
         color: ${neutralForegroundRestBehavior.var};
-        fill: ${neutralForegroundRestBehavior.var};
+        fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/menu-item/menu-item.styles.ts:68]:
```suggestion
        fill: currentcolor;
```

File: packages/web-components/fast-components/src/menu-item/menu-item.styles.ts [hunk 7]
```diff
@@ -131,7 +131,7 @@ export const MenuItemStyles = css`
                 border-color: ${SystemColors.ButtonText};
                 box-shadow: 0 0 0 calc(var(--focus-outline-width) * 1px) inset ${SystemColors.HighlightText};
                 color: ${SystemColors.HighlightText};
-                fill: ${SystemColors.HighlightText};
+                fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/menu-item/menu-item.styles.ts:134]:
```suggestion
                fill: currentcolor;
```

File: packages/web-components/fast-components/src/menu-item/menu-item.styles.ts [hunk 8]
```diff
@@ -140,7 +140,7 @@ export const MenuItemStyles = css`
             :host(.disabled:hover)::slotted(svg) {
                 background: ${SystemColors.Canvas};
                 color: ${SystemColors.GrayText};
-                fill: ${SystemColors.GrayText};
+                fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/menu-item/menu-item.styles.ts:143]:
```suggestion
                fill: currentcolor;
```

File: packages/web-components/fast-components/src/slider-label/slider-label.styles.ts [hunk 9]
```diff
@@ -10,7 +10,7 @@ import {
 export const SliderLabelStyles = css`
     ${display("block")} :host {
         color: ${neutralForegroundRestBehavior.var};
-        fill: ${neutralForegroundRestBehavior.var};
+        fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/slider-label/slider-label.styles.ts:13]:
```suggestion
        fill: currentcolor;
```

File: packages/web-components/fast-components/src/slider/slider.styles.ts [hunk 10]
```diff
@@ -59,7 +59,7 @@ export const SliderStyles = css`
         width: calc(var(--thumb-size) * 1px);
         transition: all 0.2s ease;
         color: ${neutralForegroundRestBehavior.var};
-        fill: ${neutralForegroundRestBehavior.var};
+        fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/slider/slider.styles.ts:62]:
```suggestion
        fill: currentcolor;
```

File: packages/web-components/fast-components/src/tabs/tab/tab.styles.ts [hunk 11]
```diff
@@ -32,7 +32,7 @@ export const TabStyles = css`
         height: calc(${heightNumber} * 1px);
         padding: calc(var(--design-unit) * 5px) calc(var(--design-unit) * 4px);
         color: ${neutralForegroundHintBehavior.var};
-        fill: ${neutralForegroundHintBehavior.var};
+        fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/tabs/tab/tab.styles.ts:35]:
```suggestion
        fill: currentcolor;
```

File: packages/web-components/fast-components/src/tabs/tab/tab.styles.ts [hunk 12]
```diff
@@ -43,30 +43,30 @@ export const TabStyles = css`
 
     :host(:hover) {
         color: ${neutralForegroundHoverBehavior.var};
-        fill: ${neutralForegroundHoverBehavior.var};
+        fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/tabs/tab/tab.styles.ts:46]:
```suggestion
        fill: currentcolor;
```

File: packages/web-components/fast-components/src/tabs/tab/tab.styles.ts [hunk 13]
```diff
@@ -43,30 +43,30 @@ export const TabStyles = css`
...
     }
 
     :host(:active) {
         color: ${neutralForegroundActiveBehavior.var};
-        fill: ${neutralForegroundActiveBehavior.var};
+        fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/tabs/tab/tab.styles.ts:51]:
```suggestion
        fill: currentcolor;
```

File: packages/web-components/fast-components/src/tabs/tab/tab.styles.ts [hunk 14]
```diff
@@ -43,30 +43,30 @@ export const TabStyles = css`
...
     }
 
     :host([aria-selected="true"]) {
         background: ${neutralFillRestBehavior.var};
         color: ${accentForegroundRestBehavior.var};
-        fill: ${accentFillRestBehavior.var};
+        fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/tabs/tab/tab.styles.ts:57]:
```suggestion
        fill: currentcolor;
```

File: packages/web-components/fast-components/src/tabs/tab/tab.styles.ts [hunk 15]
```diff
@@ -43,30 +43,30 @@ export const TabStyles = css`
...
     }
 
     :host([aria-selected="true"]:hover) {
         background: ${neutralFillHoverBehavior.var};
         color: ${accentForegroundHoverBehavior.var};
-        fill: ${accentFillHoverBehavior.var};
+        fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/tabs/tab/tab.styles.ts:63]:
```suggestion
        fill: currentcolor;
```

File: packages/web-components/fast-components/src/tabs/tab/tab.styles.ts [hunk 16]
```diff
@@ -43,30 +43,30 @@ export const TabStyles = css`
...
     }
 
     :host([aria-selected="true"]:active) {
         background: ${neutralFillActiveBehavior.var};
         color: ${accentForegroundActiveBehavior.var};
-        fill: ${accentFillActiveBehavior.var};
+        fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/tabs/tab/tab.styles.ts:69]:
```suggestion
        fill: currentcolor;
```

File: packages/web-components/fast-components/src/tabs/tab/tab.styles.ts [hunk 17]
```diff
@@ -120,19 +120,19 @@ export const TabStyles = css`
                 forced-color-adjust: none;
                 border-color: transparent;
                 color: ${SystemColors.ButtonText};
-                fill: ${SystemColors.ButtonText};
+                fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/tabs/tab/tab.styles.ts:123]:
```suggestion
                fill: currentcolor;
```

File: packages/web-components/fast-components/src/tabs/tab/tab.styles.ts [hunk 18]
```diff
@@ -120,19 +120,19 @@ export const TabStyles = css`
...
             }
             :host(:hover),
             :host(.vertical:hover),
             :host([aria-selected="true"]:hover) {
                 background: ${SystemColors.Highlight};
                 color: ${SystemColors.HighlightText};
-                fill: ${SystemColors.HighlightText};
+                fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/tabs/tab/tab.styles.ts:130]:
```suggestion
                fill: currentcolor;
```

File: packages/web-components/fast-components/src/tabs/tab/tab.styles.ts [hunk 19]
```diff
@@ -120,19 +120,19 @@ export const TabStyles = css`
...
             }
             :host([aria-selected="true"]) {
                 background: ${SystemColors.HighlightText};
                 color: ${SystemColors.Highlight};
-                fill: ${SystemColors.Highlight};
+                fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/tabs/tab/tab.styles.ts:135]:
```suggestion
                fill: currentcolor;
```

File: packages/web-components/fast-components/src/text-field/text-field.styles.ts [hunk 20]
```diff
@@ -83,7 +83,7 @@ export const TextFieldStyles = css`
         } width: 16px;
         height: 16px;
         margin: auto;
-        fill: ${neutralForegroundRestBehavior.var};
+        fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/text-field/text-field.styles.ts:86]:
```suggestion
        fill: currentcolor;
```

File: packages/web-components/fast-components/src/text-field/text-field.styles.ts [hunk 21]
```diff
@@ -158,7 +158,7 @@ export const TextFieldStyles = css`
             }
             .start,
             .end {
-                fill: ${SystemColors.ButtonText};
+                fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/text-field/text-field.styles.ts:161]:
```suggestion
                fill: currentcolor;
```

File: packages/web-components/fast-components/src/tree-item/tree-item.styles.ts [hunk 22]
```diff
@@ -134,7 +134,7 @@ export const TreeItemStyles = css`
         transition: transform 0.1s linear;
         ${/* transform needs to be localized */ ""} transform: rotate(-45deg);
         pointer-events: none;
-        fill: ${neutralForegroundRestBehavior.var};
+        fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/tree-item/tree-item.styles.ts:137]:
```suggestion
        fill: currentcolor;
```

File: packages/web-components/fast-components/src/tree-item/tree-item.styles.ts [hunk 23]
```diff
@@ -143,7 +143,7 @@ export const TreeItemStyles = css`
             replace when glyph-size var is added */ ""
         } width: 16px;
         height: 16px;
-        fill: ${neutralForegroundRestBehavior.var};
+        fill: currentColor;
```
Comment (janechu) [inline packages/web-components/fast-components/src/tree-item/tree-item.styles.ts:146]:
```suggestion
        fill: currentcolor;
```

File: packages/web-components/fast-components/src/flipper/flipper.styles.ts (hunk 1 above)
Comment (chrisdholt) [reply packages/web-components/fast-components/src/flipper/flipper.styles.ts:30]: +1 to all of these, but it does look like the browser normalizes here, which is likely why it's also correct upon testing for @brookdozer :).

Comment (brookdozer) [reply packages/web-components/fast-components/src/flipper/flipper.styles.ts]: Updated to all to currentcolor.

**Review by janechu — APPROVED** [review]

**Pull request merged**